src/ms-review.tex

```diff
@@ -510,121 +489,149 @@ \section{Gamma-ray Data Analysis}
-%% We also said in the abstract that Gammapy is compliant with other specifications
-%% adopted in high-energy astrophysics.
-The counts data and the reduced IRFs in the form of maps are bundled into datasets
-that represent the fourth data level (DL4). These reduced datasets can be written to disk,
-in a format specific to \gammapy to allow users to read them back at any time later
-for modelling and fitting. Different variations of such datasets support different 
-analysis methods and fit statistics. The datasets can be used to perform a joint-likelihood
-fit, allowing one to combine different measurements, e.g. from different observations
-but also from different instruments or event classes. They can also be used for binned
-simulation as well as event sampling to simulate DL3 events data.
-
-The next step is then typically to model and fit the datasets, either
-individually, or in a joint likelihood analysis. For this purpose \gammapy
-provides a uniform interface to multiple fitting backends. In addition to
-providing a variety of built-in models, including spectral,
-spatial and temporal model classes to describe the \gammaray emission in the sky,
-custom user-defined models are also supported.
-Spectral models can be simple analytical models or more complex ones from radiation
-mechanisms of accelerated particle populations (e.g. inverse Compton or $\pi^{0}$ decay).
-Independently or subsequently to the global modelling, the data can be
-re-grouped to compute flux points, light curves and flux maps as well as significance
-maps in different energy bands.
+\enquote{safe} data in the spatial as well as energy dimension.
+
+All the code related to reduce the DL3 data to binned data structures 
+is contained in the \code{gammapy.makers} sub-package.
+
+
+ \subsubsection{DL4: Binned Data Structures}
+ The counts data and the reduced IRFs in the form of maps are bundled into datasets
+ that represent the fourth data level (DL4). These reduced datasets can be written to disk,
+ in a format specific to \gammapy to allow users to read them back at any time later
+ for modelling and fitting. Different variations of such datasets support different 
+ analysis methods and fit statistics. The datasets can be used to perform a joint-likelihood
+ fit, allowing one to combine different measurements, e.g. from different observations
+ but also from different instruments or event classes. They can also be used for binned
+ simulation as well as event sampling to simulate DL3 events data.
+
+
+Binned maps and datasets, which represent a collection of binned
+maps, are defined in the \code{gammapy.maps} and \code{gammapy.datasets}
+sub-packages, respectively. 
+
+
+\subsection{From DL4 to DL5/6: Modeling and Fitting}
+The next step is then typically to model the datasets using binned Poisson maximum likelihood fitting.
+Assuming Poisson statistics per bin the log-likelihood of an observation is given by the Cash statistics\citep{Cash}:
```

Comment:
Since there are two possible likelihoods, I would avoid giving the explicit Cash expression but simply mention 
$\log L(N_{obs, i}| N_{pred, i}$ and possibly cite Cash as an example.

Reply:
I see you point, but I think the Cash statistics is the more fundamental and important one. With Gammapy we have made the transition from the "classical methods" to more modern ones, which are typically based on background templates. The community will follow...